frontend/src/services/SpotifyAPI/component.ts

```diff
@@ -0,0 +1,86 @@
+import { type trackInterface } from '@pages/Feed/Post'
+import { generateRandomUID } from '@services/IdGenertor'
+import { type components } from '@data/spotify-types'
+type trackObject = components['schemas']['TrackObject']
+type PrivateUserObject = components['schemas']['PrivateUserObject']
+
+export async function getTrack(trackId: string): Promise<trackInterface> {
+  const bearer = window.localStorage.getItem('access_token') ?? ''
+
+  if (bearer === '') {
+    console.error('Token was empty string')
+  }
+
+  const optionsTrack = {
+    method: 'GET',
+    headers: {
+      Authorization: `Bearer ${bearer}`,
+    },
+  }
+
+  return await request<trackObject>(
+    `https://api.spotify.com/v1/tracks/${trackId}`,
+    optionsTrack,
+  )
+    .then((res) => {
```

Comment:
Default Value " t " not so clean? 
Maybe set title -> Title, artist -> Artist.... ?